apps/io/feeding_service_webhook.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2016 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+from superdesk.io.commands import update_ingest
+from superdesk.io.registry import registered_webhooks
+import superdesk
+
+
+class FeedingServiceWebhookResource(Resource):
+    public_methods = ['GET', 'POST']
```

Comment:
@petrjasek I'm not sure what you mean with auth token, the requesting entity (wufoo) will not provide it. By "more efficient way", I mean that if somebody really want to do a DoS, he'll better go with a DDoS tool like LOIC than guessing a specific url (which is not supposed to be public, even if for NTB it's currently hardcoded) to trigger ingestion.

The issue by using handshake or other shared secret is that it is vendor specific and not generic anymore. A hidden URL is a kind of shared secret anyway (we can autogenerate a key in the URL). An yes HTTPS is needed.

Maybe we can talk about this during the scrum?